apps/backend/src/common/entities/FixedWidthRecord.ts

```diff
@@ -132,9 +139,9 @@ export class FixedWidthRecord<T extends IFixedWidthRecord<T>>
     if (target.delimiter?.value) {
       target.delimiter.positions.forEach((pos: number) => {
         result = Buffer.concat([
-          result.slice(0, pos + 1),
+          result.subarray(0, pos + 1),
```

Comment:
Why this change? I think it also works, but I'm curious

Reply:
according to sonarqube slice is deprecated 🤷‍♀️